src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -34,4 +42,12 @@ public class Book {
     private String coverImage;
     @Column(nullable = false, columnDefinition = "TINYINT", length = 1)
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToMany
```
It's the default fetch type for this type of relation ;)